### PR TITLE
Check if processed data already exists in --production mode

### PR DESCRIPTION
**Pull request by mflierm**
## What does the code in this PR do / what does it improve?
The code checks in the runs database if the run that is being tried to process, has already been processed  (i.e. `bootstrax_status in ['done', 'processing', 'considering']`). This avoids that the processing results in `'failed'` (because there is no `live_data` available anymore).

## Can you briefly describe how it works?
See above.

## Can you give a minimal working example (or illustrate with a figure)?

_Please include the following if applicable:_
  - [ ] _Update the docstring(s)_
  - [ ] _Update the documentation_
  - [ ] _Tests to check the (new) code is working as desired._
  - [x] _Does it solve one of the open issues on github?_

### _Notes on testing_
 - _Until the automated tests pass, please mark the PR as a draft._
 - _On the XENONnT fork we test with database access, on private forks there is no database access for security considerations._

### CODE REVIEW

File: bin/bootstrax [hunk 1]
```diff
@@ -345,11 +345,20 @@ def main():
     elif args.process:
         t_start = now()
         number = args.process
+
+        # Check whether the run is already processed
+        bootstrax_state = run_col.find_one({'number': number}, projection={'bootstrax': True}).get('bootstrax', {}).get{'state', 'no-state')
```
Comment (JoranAngevaare) [inline bin/bootstrax:350]:
```suggestion
        bootstrax_state = run_coll.find_one(
            {'number': number},
            projection={'bootstrax': True}).get('bootstrax', {}
                                                ).get('state', 'no-state')
```
couple of typos

File: bin/bootstrax [hunk 2]
```diff
@@ -345,11 +345,20 @@ def main():
...
+        if args.production and bootstrax_state in ['done','processing','considering']:
```
Comment (JoranAngevaare) [inline bin/bootstrax:351]:
```suggestion
        if args.production and bootstrax_state in ['done', 'busy', 'considering']:
```

**Review by JoranAngevaare — COMMENTED** [review]
Thanks Maricke, I think this is very useful (I at least screwed this up a couple of times)! Looks good. Let's make sure we test it once just before making the next release - just to be sure

File: bin/bootstrax (hunk 2 above)
Comment (mflierm) [reply bin/bootstrax]: Done! :)

**Review by JoranAngevaare — APPROVED** [review]
Thanks a lot @mflierm - I'll merge it after we have done a final test!

**Comment by JoranAngevaare** [comment]
Nice work Maricke!

**Pull request merged**